README.md

```diff
@@ -135,6 +135,45 @@ steps:
 
 > Note: The `id` defined in the `download/artifact` step must match the `id` defined in the `echo` step (i.e `steps.[ID].outputs.download-path`)
 
+# Waiting for the artifact to be available
+
+You can specify `waitTimeout` (seconds) to instruct the download/artifact to retry until the artifact is available.
```

Comment:
Maybe rename the input to `wait-timeout` to fit with other options?

Reply:
thx, good point -> renamed.